### PR TITLE
Styles application issues

### PR DESCRIPTION
**Pull request by vpereverzev**
resolves - https://trello.com/c/sabysJAY

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -3560,15 +3560,11 @@ static void loadScores(const QStringList& argv)
...
                         continue;
-                  MasterScore* score = mscore->readScore(name);
-                  if (score) {
-                        mscore->appendScore(score);
+
+                  Score* score = mscore->openScore(name);
+
+                  if (score)
                         scoresOnCommandline = true;
-                        if(!MScore::noGui) {
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp:3567]: Isn't double-clicking a score also affected by this? It is on the command line in that case, but in GUI mode, as far as I can  tell

Comment (vpereverzev) [reply mscore/musescore.cpp:3567]: Yep, I didn't mention it in the commit message, but it also affect such a case.

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:3567]: But then it should add the score to the list of recently used ones, shouldn't it? Or does it still?

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:3567]: Also while at it, you could change that `foreach()` ;-)

**Comment by Jojo-Schmitz** [comment]
Strange Mac build fail, doesn't seem to be related to this PR but rather a general issue?

File: mscore/musescore.cpp (hunk 1 above)
Comment (vpereverzev) [reply mscore/musescore.cpp:3567]: the openscore method already does all this, so I just cleaned it up here as unnecessary

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:3567]: I see, sorry for the noise then

**Comment by vpereverzev** [comment]
> Strange Mac build fail, doesn't seem to be related to this PR but rather a general issue? See https://github.com/musescore/MuseScore/pull/7116/checks#step:5:43

there is a known issue with python installation via brew, we're trying to figure out how to resolve this

**Comment by MarcSabatella** [comment]
Currently it seems the style settings don't work correctly on the command line.  If I explicitly set no first system indent in a 3.6 score, command line PNG conversion nevertheless adds an indent.  This show up in the new vtests I tried adding for PR #7110.  I can't tell if this fix will address that also.  But I think so?

**Comment by vpereverzev** [comment]
> 
> 
> Currently it seems the style settings don't work correctly on the command line. If I explicitly set no first system indent in a 3.6 score, command line PNG conversion nevertheless adds an indent. This show up in the new vtests I tried adding for PR #7110. I can't tell if this fix will address that also. But I think so?

Yes, I think this fix would help you

**Pull request merged**